### ISSUE TITLE
Emoji history

### ISSUE THREAD

**Issue by BenBress** [dslul/openboard#773]
On the 1.4.5 version, the history of the emoji icons is not updated, even if it was the case with the 1.4.4 version

**Comment by Ammako** [comment]
If you enabled incognito mode in the settings, this is intentional behavior.

#521 it's not like I didn't warn them this would happen 🙄

Download the source, revert 6eb8134b6c3f24b25c66b7eb5a1632a1a0fec01f, and build from that modified source to get this working properly again. I'll be happy to walk you through this step-by-step if you need to.

**Comment by mister-sir** [comment]
@Ammako : yeah, after you pointed that out in #521 I realized I didn't entirely like the solution I had. Different people might want to use Incognito mode differently. I've been meaning to find the time to look at a way to put this as a toggleable setting, but I haven't had nearly as  much free time as I used to in the last year.
